tests/test_other.py

```diff
@@ -5755,6 +5755,13 @@ def test_no_warn_exported_jslibfunc(self):
     out, err = Popen([PYTHON, EMCC, path_from_root('tests', 'hello_world.c'), '-s', 'DEFAULT_LIBRARY_FUNCS_TO_INCLUDE=["alGetError"]', '-s', 'EXPORTED_FUNCTIONS=["_main", "_alGetError"]'], stdout=PIPE, stderr=PIPE).communicate()
     self.assertNotContained('''function requested to be exported, but not implemented: "_alGetError"''', err)
 
+  def test_suppress_almost_asm_warning(self):
```

Comment:
We should also have one test that passes both the `-Wfoo` and `-Wno-foo` forms of one of the flags to check that the second one overrides the first.